androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt

```diff
@@ -75,6 +75,9 @@ class ComposePrototypeUITest : BaseTests() {
     private val fakeAuthUserName = "daffy.duck@looneytunes.com"
     private val fakeAuthPassword = "xxxxxxxxxx"
     private val TAG = TestBedViewModel::class.simpleName
+    private val clearConversation = "clearConversation"
+    private val clearConversationMessage = "Connection Closed Normally"
```

Comment:
Can we, please rename this 2 properties to:
- connectionClosedMessage
- connectionClosedCode